examples/README.md

```diff
@@ -3291,24 +3341,57 @@ We fist define our top-level workflow for this example:
        },
        {
           "name": "DoCarVitalsChecks",
-          "type": "subflow",
+          "type": "operation",
+          "actions": [
+            {
+              "subFlow": {
+                "workflowId": "vitalscheck",
+                "waitForCompletion": false
+              },
+            }
+          ],
+          "transition": "WaitForCarStopped"
+       },
+       {
+          "name": "WaitForCarStopped",
+          "type": "event",
           "workflowId": "vitalscheck",
-          "repeat": {
-             "stopOnEvents": ["CarTurnedOffEvent"]
-          },
+          "onEvents": [
```

Comment:
Don't understand the logic here. The intend is to repeat execution of the subflow until the car stopped event is consumed. this seems its just doing it once and transitioning to this event state. am i missing something?

Reply:
ah ok there are more states below.